src/content/quickstarts/automation-station.mdx

```diff
@@ -0,0 +1,251 @@
+
+To register a new upkeep, edit the necessary parameters inside `scripts/registerUpkeep.ts` and run the script.
+
+1. Navigate to the `scripts/registerUpkeep.ts` file to find the upkeep registration script.
+1. Fill in the encoded `registrationParams`.
+    - The `adminAddress` must be the address of the station itself. 
+    - The station's LINK token balance must be greater than or equal to the amount you set in the `amount` parameter.
+    - The allowance of LINK tokens to the Chainlink Automation Registrar must also be greater than or equal to the `amount`.
+
+    Use the following example to encode your
+    `registrationParams`:
+
+    ```ts
+    /** RegistrationParams inputs
+        * string name = "UpkeepName";
+        * bytes encryptedEmail = 0x;
+        * address upkeepContract = 0x...; // Use your station's contract address
+        * uint32 gasLimit = 500000;
+        * address adminAddress = 0x....; // Use your station's contract address
+        * uint8 triggerType = 0;
+        * bytes checkData = 0x;
+        * bytes triggerConfig = 0x;
+        * bytes offchainConfig = 0x;
+        * uint96 amount = 2000000000000000000;
+    */
+    const registrationParams = ethers.AbiCoder.defaultAbiCoder().encode(
+    ["tuple(string,bytes,address,uint32,address,uint8,bytes,bytes,bytes,uint96)"],
+    [["UpkeepName", "0x", "0x794Db242b1d75663E2d821B0105cFd4E5Bb182F0", 1000000, "0x794Db242b1d75663E2d821B0105cFd4E5Bb182F0", 0, "0x", "0x", "0x", ethers.toBigInt("2000000000000000000")]]
+    );
+    ```
+1. Run the upkeep registration script:
+
+    ```sh
+    npx hardhat run scripts/registerUpkeep.ts --network arbitrum_sepolia
+    ```
+</Accordion>
+
+<Accordion title="Add existing upkeeps for balance monitoring" number={5}>
+
+Use the `addUpkeeps` function to add a list of existing upkeeps for your AutomationStation to monitor. This function does not directly modify any of your upkeeps in Chainlink Automation; instead, it adds upkeep IDs to the AutomationStation's `s_upkeepIds` list, which is then used for balance monitoring. 
+
+1. Navigate to the `scripts/addUpkeeps.ts` file.
+1. Fill in the address of your deployed AutomationStation contract and the list of upkeep IDs for the station to monitor.
+1. Save your changes to the `scripts/addUpkeeps.ts` file.
+1. Run the script:
+
+    ```sh
+    npx hardhat run scripts/addUpkeeps.ts --network arbitrum_sepolia
+    ```
+This script does not display any output, but you can verify that your upkeep IDs were added by checking the value of `s_upkeepIds`, or viewing your deployed station contract in the block explorer to confirm that the `addUpkeeps` function ran successfully.
```

Comment:
Should we add a step to verify the contract on Etherscan? The project is already set up to do so (@nomicfoundation/hardhat-verify).

Reply:
Good catch on the hardhat-verify library, but I'm not sure how that fits within this "add upkeeps" section. We're adding existing upkeeps to the station watchlist, so that's after those contracts have already been deployed